src/main/java/ua/danit/final_project/controllers/WorkShiftController.java

```diff
@@ -0,0 +1,74 @@
+package ua.danit.final_project.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import ua.danit.final_project.entities.ShiftComment;
+import ua.danit.final_project.entities.WorkShift;
+import ua.danit.final_project.services.WorkCommentService;
+import java.net.URI;
+import java.sql.Timestamp;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.sql.Timestamp'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p8HX2sY87p04Y7_&open=AWWG_p8HX2sY87p04Y7_)

[ISSUE_KEY]: AWWG_p8HX2sY87p04Y7_